webapp/src/marked.tsx

```diff
@@ -284,6 +285,19 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
         }
     }
 
+    private renderVideo(content: HTMLElement){
+        pxt.Util.toArray(content.querySelectorAll('Video.ams-embed'))
+            .forEach((inlineVideo: HTMLElement)=> {
+                let player = MediaPlayer().create()
+                if (inlineVideo) {
+                    player.initialize(inlineVideo, inlineVideo.getAttribute("src"));
+                }
+                return () => {
```

Comment:
```suggestion
                return () => {
```

this doesn't look like it's doing anything?